src/OpenApi/test/Microsoft.AspNetCore.OpenApi.Tests/Integration/snapshots/OpenApiDocumentIntegrationTests.VerifyOpenApiDocument_documentName=controllers.verified.txt

```diff
@@ -90,10 +88,7 @@
       }
     }
   },
-  "components": { },
   "tags": [
-    {
-      "name": "Test"
-    }
+    "Test"
```

Comment:
This does not look right. The top-level `Tags` field should be an array of `TagObject`s, not `string`s.

Reply:
Yep, this is a known bug that I filed on the library over at https://github.com/microsoft/OpenAPI.NET/issues/1991. It was fixed but the version with the fix contains a different bug https://github.com/microsoft/OpenAPI.NET/issues/2055 so I am sticking with the devil I know for now. 😅